test/integration/universe/wizards-v1/universe-lifecycle/basic-paths/scenario-2.ts

```diff
@@ -0,0 +1,257 @@
+import { UniverseValidatorV1 } from '../../../../../validators/wizards-v1/universe/universe-validator-v1';
+
+export const SCENARIO_2 = 'scenario-2';
+
+export const scenario2 = (): void => {
+  /**** Core Contracts ****/
+  let universeRegistry: IUniverseRegistry;
+  let universeWizardV1: IUniverseWizardV1;
+  let warperPresetFactory: IWarperPresetFactory;
+  let warperManager: IWarperManager;
+  let taxTermsRegistry: ITaxTermsRegistry;
+  let metahub: IMetahub;
+  /**** Mocks & Samples ****/
+  let baseToken: ERC20Mock;
+  let originalCollection: ERC721Mock;
+  /**** Signers ****/
+  let universeOwner: SignerWithAddress;
+  let stranger: SignerWithAddress;
+  /**** Data Structs ****/
+  let universeConstructor: UniverseConstructorV1;
+  let warperConstructor: Erc721ConfigurableWarperConstructor;
+  const taxTerms = makeTaxTermsFixedRateFromRawPercent('5');
+
+  beforeEach(function () {
+    universeRegistry = this.contractsNew.universeRegistry;
+    universeWizardV1 = this.contractsNew.wizardsV1.universeWizard;
+    warperPresetFactory = this.contractsNew.warperPresetFactory;
+    warperManager = this.contractsNew.warperManager;
+    taxTermsRegistry = this.contractsNew.taxTermsRegistry;
+    metahub = this.contractsNew.metahub;
+    originalCollection = this.mocksAndSamples.assets.erc721.originalCollections.A;
+
+    baseToken = this.mocksAndSamples.assets.erc20.baseToken;
+
+    [universeOwner, stranger] = this.signersNew.unnamed;
+
+    warperConstructor = new Erc721ConfigurableWarperConstructor()
+      .withOriginalCollection(originalCollection)
+      .withMetahub(metahub);
+    universeConstructor = new UniverseConstructorV1()
+      .withUniverseName(COMMON_TEST_UNIVERSE_NAME_1)
+      .withPaymentToken(baseToken);
+  });
+
+  it(`${TEST_CASE(TEST_OK)} Setup Universe with a Warper deployed from preset`, async () => {
+    const universeBuilder = new UniverseWithErc721WarperBuilderV1(universeWizardV1, universeOwner)
+      .withUniverseConstructor(universeConstructor)
+      .withWarperConstructor(warperConstructor)
+      .withTaxTerms(taxTerms);
+    const { setupUniverseSuccessTx, universeId, deployedWarperAddress } =
+      await universeBuilder.setupUniverseWithSuccessAndTx();
+
+    await UniverseValidatorV1.validateUniverseSetupProperly(
+      setupUniverseSuccessTx,
+      universeId,
+      universeOwner,
+      universeConstructor.getUniverseParamsSafely(),
+      {
+        universeRegistry,
+        reader: stranger,
+      },
+    );
+    await WarperPresetFactoryValidator.validateProperDeployFromWarperPreset(
+      setupUniverseSuccessTx,
+      deployedWarperAddress,
+      ERC721_WARPER_PRESET_IDS.ERC721_CONFIGURABLE_PRESET,
+      {
+        warperPresetFactory,
+      },
+    );
+    await Erc721ConfigurablePresetValidator.validateWarperState(
+      deployedWarperAddress,
+      originalCollection,
+      {
+        availabilityPeriodStart: 0,
```

Comment:
Myself forgot to use ZERO in one other example, could you use here the ZERO constant and also check other place, where the same situation (search for vailabilityPeriodStart: 0)